recipes-bsp/uefi/uefi_git.bb

```diff
@@ -5,8 +5,8 @@ LIC_FILES_CHKSUM = "file://NXP-Binary-EULA;md5=343ec8f06efc37467a6de53686fa6315"
 
 inherit deploy
 
-SRC_URI = "git://github.com/NXP/qoriq-uefi-binary.git;nobranch=1"
-SRCREV= "cd84334952594b241f331829d6a092ffe971eb2d"
+SRC_URI = "git://github.com/NXP/qoriq-uefi-binary;protocol=https;nobranch=1"
```

Comment:
The protocol change is not needed as git protocol also works on github.

Reply:
drop protocol change now.